### ISSUE TITLE
Insert title and desc tags

### ISSUE THREAD

**Issue by syuji-higa** [nuxt-community/svg-sprite-module#10]
Want to insert title and desc tags.

See follow the example.

Set title and desc tags to svg element.

```html
<svg-icon
  name="sample"
  title="Title"
  desc="Description"
/>
```

After built,

```html
<svg xmlns="http://www.w3.org/2000/svg">
  <title>Title</title>
  <desc>Description</desc>
  <use xlink:href="/_nuxt/assets/sprite/gen/icons.svg#i-sample"></use>
</svg>
```

**Comment by farnabaz** [comment]
Hi
Seems nice, as [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) says:  
> The `<title>` element generally improves accessibility of SVG documents.

We will consider this in our next release, Thank you

**Comment by farnabaz** [comment]
🎉 This issue has been resolved in version 0.2.0 🎉
